apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,355 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  AvatarBaseShape,
+  AvatarToken,
+  AvatarTokenSize,
+  BadgeNetwork,
+  ButtonSize,
+  ButtonIcon,
+  ButtonIconSize,
+  IconName,
+  Text,
+  TextVariant,
+  TextColor,
+  AvatarAccount,
+  AvatarAccountVariant,
+  Icon,
+  ButtonBase,
+  Box,
+  BoxBackgroundColor,
+  BoxBorderColor,
+} from '@metamask/design-system-react';
+
+const meta: Meta = {
+  title: 'Examples/Wallet Home',
+  component: () => null,
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+const WalletHome: React.FC = () => {
+  return (
+    <div className="bg-background-default min-h-screen">
+      {/* Header */}
+      <Box
+        backgroundColor={BoxBackgroundColor.BackgroundDefault}
+        padding={4}
+        borderColor={BoxBorderColor.BorderMuted}
+      >
+        <div className="flex items-center justify-between">
+          <div className="flex items-center space-x-2">
+            <AvatarAccount
+              shape={AvatarBaseShape.Square}
+              variant={AvatarAccountVariant.Maskicon}
+              address=""
+            />
+            <Text variant={TextVariant.HeadingMd} color={TextColor.TextDefault}>
+              DeFi Account
+            </Text>
+            <ButtonIcon
+              iconName={IconName.ArrowDown}
+              size={ButtonIconSize.Sm}
+              ariaLabel="Switch Account"
+            />
+          </div>
+          <div className="flex items-center space-x-4">
+            <ButtonIcon
+              iconName={IconName.Copy}
+              size={ButtonIconSize.Md}
+              ariaLabel="Copy"
+            />
+            <ButtonIcon
+              iconName={IconName.Global}
+              size={ButtonIconSize.Md}
+              ariaLabel="Network"
+            />
+            <ButtonIcon
+              iconName={IconName.Menu}
+              size={ButtonIconSize.Md}
+              ariaLabel="Menu"
+            />
+          </div>
+        </div>
+      </Box>
+
+      {/* Balance */}
+      <div className="p-4">
+        <Text variant={TextVariant.DisplayMd} color={TextColor.TextDefault}>
+          $10,528.46
+        </Text>
+        <div className="flex items-center space-x-2">
+          <Text variant={TextVariant.BodyMd} color={TextColor.SuccessDefault}>
+            ↑
+          </Text>
+          <Text variant={TextVariant.BodyMd} color={TextColor.SuccessDefault}>
+            $367.00 (+0.66%)
+          </Text>
+        </div>
+      </div>
+
+      {/* Actions */}
+      <div className="grid grid-cols-4 gap-2 p-4">
+        <ButtonBase
+          size={ButtonSize.Md}
+          className="bg-muted bg-prima hover:bg-muted-hover active:bg-muted-pressed h-auto flex-col justify-center rounded-lg p-4"
+        >
```

Comment:
I see, so we'd still be able to use the component library props, but also use tailwind to do any sort of custom style overrides.

Reply:
I wonder if there is a way to enforce a consistent behavior across the app? How will we know that a developer isn't just freestyling the styles here and making sure they follow the designs as they were intended?